GameEngine/Commands/LookCommand.cs

```diff
@@ -76,7 +76,8 @@ public void Execute(List<string> extraWords, Character lookingCharacter)
                 }
             }
 
-            var otherCharactersInLocation = GameState.CurrentGameState.GetCharactersInLocation(lookingCharacterLocation.TrackingId, includePlayer: false);
+            var otherCharactersInLocation = GameState.CurrentGameState.GetCharactersInLocation(lookingCharacterLocation.TrackingId, includePlayer: true);
```

Comment:
I think we need to refactor this method to accept a list of excludes instead of just a bool.

Reply:
you can't expect me to know everthing, so you tell me what are excludes.